File: PRAK603-2510817220023-NURFITRIA RAMADHANI.c
#include <stdio.h>

int main() {
    int a,b;
    scanf("%d %d", &a, &b);
    if(a!=b){
        printf("Jumlah tidak sama");
        return 0;
    }
    int x[a], y[b];
    for(int i=0;i<a;i++){
        scanf("%d",&x[i]);
    }
    for(int i=0;i<b;i++){
        scanf("%d",&y[i]);
    }
    for(int i=0;i<a;i++){
        printf("%d ",x[i]*y[i]);
    }
    return 0;
}

File: PRAK603-2510817220023-NURFITRIA RAMADHANI.py
a, b = map(int, input().split())

if a != b:
    print("Jumlah tidak sama")
else:
    x = list(map(int, input().split()))
    y = list(map(int, input().split()))

    for i in range(a):
        print(x[i] * y[i], end=" ")